config/environments/production.rb

```diff
@@ -46,7 +46,7 @@
   # config.action_cable.allowed_request_origins = [ 'http://example.com', /http:\/\/example.*/ ]
 
   # Force all access to the app over SSL, use Strict-Transport-Security, and use secure cookies.
-  # config.force_ssl = true
+  config.force_ssl = ScihistDigicoll::Env.lookup!(:force_ssl) == "true"
```

Comment:
If it's boolean, then `== "true"` will never be true I think? Because `true == "true"` is never true? Have you tested this? I think it just needs to be `config.force_ssl = ScihistDigicoll::Env.lookup!(:force_ssl)`, no? And maybe test to make sure it's working! Thanks!

Reply:
No, ScihistDigicoll::Env.lookup!(:force_ssl) returns a string, not a boolean. I did test it.

Reply:
OH I see what's going on. Never mind.